CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Contributing Guidelines
```

Comment:
For what it's worth, I generally use Contributor Covenant and modify https://github.com/vendia/serverless-express/blob/mainline/CODE_OF_CONDUCT.md

Reply:
Good call - added in update.